server_utils/origin_test.go

```diff
@@ -214,6 +223,162 @@ func TestGetExports(t *testing.T) {
 		assert.False(t, viper.GetBool("Origin.EnableListings"))
 		assert.True(t, viper.GetBool("Origin.EnableDirectReads"))
 	})
+
+	// S3 tests
+	t.Run("testSingleExportValidS3", func(t *testing.T) {
```

Comment:
Why was the function name changed to TestGetExportsPosix when there are S3 tests in it?

Reply:
Because I started writing the tests differently and then decided to consolidate but forgot to change it back 🙃 Easy enough to fix though!